packages/mockserver-config-writer/src/mockserver-config/ui5-mock-yaml.ts

```diff
@@ -37,13 +37,18 @@ export async function enhanceYaml(
     let mockConfig;
     const manifest = fs.readJSON(join(webappPath, 'manifest.json')) as Partial<Manifest> as Manifest;
     const mockserverPath = config?.path || getMainServiceDataSource(manifest)?.uri;
+    const annotations = getMainServiceDataSource(manifest)?.settings?.annotations || [];
```

Comment:
Lets have a chat if this should be filtered by type i.e. `ODataAnnotation` or by reading the mainService annotations list.